tests/spread/charms/operator/task.yaml

```diff
@@ -5,9 +5,29 @@ systems:
 kill-timeout: 30m
 
 environment:
-  CHARM/mysql: https://github.com/canonical/mysql-operator
+  CHARM/cos_proxy: https://github.com/canonical/cos-proxy-operator
+  CHARM/dex_auth: https://github.com/canonical/dex-auth-operator
+  CHARM/github_runner: https://github.com/canonical/github-runner-operator
+  CHARM/hardware_observer: https://github.com/canonical/hardware-observer-operator
+  CHARM/hydra: https://github.com/canonical/hydra-operator
+  CHARM/identity_platform: https://github.com/canonical/identity-platform-login-ui-operator
+  CHARM/indico: https://github.com/canonical/indico-operator
+  CHARM/kafka: https://github.com/canonical/kafka-operator
+  CHARM/kratos: https://github.com/canonical/kratos-operator
+  CHARM/minio: https://github.com/canonical/minio-operator
+  CHARM/mlflow: https://github.com/canonical/mlflow-operator
+  CHARM/mongodb: https://github.com/canonical/mongodb-operator
+  # Disabled because it uses a tox-based workflow
+  # CHARM/mysql: https://github.com/canonical/mysql-operator
+  # Doesn't use charmcraft
+  # CHARM/nats: https://github.com/canonical/nats-operator
```

Comment:
Any reason to keep this?

Reply:
TODOs for later this pulse (same as above).